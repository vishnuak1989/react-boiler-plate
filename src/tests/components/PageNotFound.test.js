import React from 'react'
import  PageNotFound  from '../../components/PageNotFound'
// import Reactshallowrenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme'

test("Should render  Expense Dashboard Page",()=>{
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
})
