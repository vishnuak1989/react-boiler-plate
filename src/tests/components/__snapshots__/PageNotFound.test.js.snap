// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render  Expense Dashboard Page 1`] = `
<div>
  <p>
    Page Not Found
  </p>
  <Link
    replace={false}
    to="/"
  >
     Go Home
  </Link>
</div>
`;
